src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml

```diff
@@ -104,6 +104,42 @@
     </StringProperty.ValueEditors>
   </StringProperty>
 
+  <!-- TODO use fwlink for HelpUrl -->
+  <BoolProperty Name="UseWindowsForms"
+                DisplayName="Windows Forms"
+                Description="Enable Windows Forms for this project."
+                HelpUrl="https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props-desktop"
```

Comment:
https://review.docs.microsoft.com/help/contribute/links-how-to?branch=master#locale-codes-in-external-links
```suggestion
                HelpUrl="https://docs.microsoft.com/dotnet/core/project-sdk/msbuild-props-desktop"
```

Reply:
This is awesome, thanks. Fixed in this PR, and I also filed https://github.com/dotnet/project-system/pull/7283 to fix it in a few other places. Appreciated.